### PR TITLE
fix(dlm): opt error message when editing the limit configuration

### PR DESCRIPTION
**Pull request by guowl3**
#### What type of PR is this?
type-feature

#### What this PR does / why we need it:
FYI
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by kiko-art — APPROVED** [review]
lgtm

**Review by yhilmare — APPROVED** [review]
lgtm

File: server/odc-core/src/main/resources/i18n/BusinessMessages.properties [hunk 1]
```diff
@@ -142,6 +142,9 @@ com.oceanbase.odc.LimitMetric.WORKSHEET_CHANGE_COUNT=The number of changed works
 com.oceanbase.odc.LimitMetric.WORKSHEET_SAME_LEVEL_COUNT=The number of worksheets at the same level
 com.oceanbase.odc.LimitMetric.WORKSHEET_COUNT_IN_PROJECT=The number of worksheets in the project
 com.oceanbase.odc.LimitMetric.WORKSPACE_COUNT_IN_PROJECT=The number of workspaces in the project
+com.oceanbase.odc.LimitMetric.DLM_ROW_LIMIT=row limit
+com.oceanbase.odc.LimitMetric.DLM_DATA_SIZE_LIMIT=data size limit
```
Comment (JessieWuJiexi) [inline server/odc-core/src/main/resources/i18n/BusinessMessages.properties:146]: 按照文案规范，首字母建议大写，Row limit, Data size limit

**Review by yhilmare — APPROVED** [review]
lgtm

**Review by JessieWuJiexi — APPROVED** [review]

**Review by kiko-art — APPROVED** [review]
lgtm

**Review by yizhouxw — APPROVED** [review]

**Pull request merged**